src/js/Toolbar/L.PM.Toolbar.js

```diff
@@ -563,7 +563,16 @@ const Toolbar = L.Class.extend({
     this.changeControlOrder();
     return control;
   },
-
+  controlExists(name) {
+    return Boolean(this.getButtons()[name]);
+  },
+  getCustomControls() {
+    return Object.fromEntries(
+      Object.entries(this.getButtons()).filter(
+        ([, v]) => v._button.tool === 'custom'
+      )
+    );
+  },
```

Comment:
Let us make this more general:

```suggestion
getButtonsInBlock(name) {
  const buttonsInBlock = {};
    if(name){
    for(const buttonName in map.pm.Toolbar.getButtons()){
      const button = map.pm.Toolbar.getButtons()[buttonName];
      // draw controls doesn't have a block
      if(button._button.tool === name || (name === 'draw' && !button._button.tool)){
        buttonsInBlock[buttonName] = button;
      }
    }
  }
  return buttonsInBlock;
}
```